game/script-greetings.rpy

```diff
@@ -1769,7 +1769,37 @@ init 5 python:
 label greeting_night_night_owl:
     n 1unmajesu "Oh!{w=0.3} [player]!{w=1}{nw}"
     extend 3fllsslsbl " You're a night owl too,{w=0.2} huh?"
-    n 3fcsbg "N-{w=0.2}not that I have a problem with that,{w=0.2} obviously." 
+    n 3fcsbg "N-{w=0.2}not that I have a problem with that,{w=0.2} obviously."
     extend 4nchgnl " Welcome back!"
 
     return
+
+
+# Natsuki getting caught off guard by the player (again)
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._greeting_database,
+            label="greeting_startled",
+            unlocked=True,
+                affinity_range=(jn_affinity.NORMAL, jn_affinity.LOVE),
+                ),
+            topic_group=TOPIC_TYPE_GREETING
+            )
+
+label greeting_startled:
+    n 1kcsunl "..."
+    n 1kwmunleqm "...?"
+    n 1kskemleex "A-{w=0.2}ah!"
+    n 1fsqeml "[player_initial]-{w=0.1}[player]!-{w=0.5}{nw}"
+    extend 2fcsunl " are you {w=0.2}{i}trying{/i} to give me a heart attack?!"
+    n 2ccsemlesi "But... {w=0.3}{nw}"
+    extend 1nnmssl "I'm glad to see you again..."
```

Comment:
Also we always start a line of extend with a space, so there is none spacing missing!